### DISCUSSION TITLE
Nuxt 3 works on firebase but not in dev env

### DISCUSSION THREAD

**Discussion by adityar15** [nuxt/nuxt#25052] [Questions]
I have a Nuxt 3 app.

Here are the system specs:

```
node: 21.5.0
mac: 14.2.1
vite: 5.0.0
nuxt: 3.9.0

```

On Firebase I am using Node 18.0.0 engine. 

It works smoothly on Firebase hosting. On my dev server, the navigations are broken and I am getting weird import errors. The same app was working fine on the dev server. My guess is it started to behave weirdly after I updated to Node version 21.5.0 and  Nuxt 3.3 to 3.9.

Any help is much appreciated :)

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Comment by adityar15** [comment]
Finally, I found the issue. So I have defined all the TS interfaces in assets/js/interface.ts

I was importing them as below

```
import {ArticleInterface} from "@/assets/js/interface"
```

This worked great before the update. After update it threw the error that it "could not find Article from in /_nuxt/assets/js/interface.ts

After update I need to edit the tsconfig to add the configs as below. I added them in nuxt.config.ts

```
typescript: {
    tsConfig: {
      compilerOptions: {
        verbatimModuleSyntax: false
      }
    }
  }
```

I believe the verbatimModuleSyntax is set to true in the update which I set to false. All works great like before.
